packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -930,14 +937,36 @@ ${chalk.cyan("?")} Your Prismic repository name`.replace("\n", ""),
 						"Project framework must be available through context to run `createNewRepository`",
 					);
 
-					await this.manager.prismicRepository.create({
-						domain: this.context.repository.domain,
-						framework: this.context.framework.wroomTelemetryID,
-						starterId: this.context.starterId,
-					});
+					try {
+						await this.manager.prismicRepository.create({
+							domain: this.context.repository.domain,
+							framework: this.context.framework.wroomTelemetryID,
+							starterId: this.context.starterId,
+						});
+					} catch (e) {
```

Comment:
💅 We usually name errors `error` for consistency on the backend (at least, afaik)

Reply:
You're right, and I vote for consistency!

Reply:
Done 🙂